src/app/SharpRaven/RavenClient.cs

```diff
@@ -169,6 +170,33 @@ public Dsn CurrentDsn
         /// </value>
         public TimeSpan Timeout { get; set; }
 
+        /// <summary>
+        /// Not register the <see cref="BreadcrumbsRecord"/> for tracking.
+        /// </summary>
+        public bool IgnoreBreadcrumbs { get; set; }
```

Comment:
I get this idea in another getsentry client. (I think phyton client). I think that is a good feature and we give to user the power of the make decision when the trail will be enabled or not.